app/src/main/java/org/kiwix/kiwixmobile/search/AutoCompleteAdapter.java

```diff
@@ -55,22 +58,20 @@ private void setupDagger() {
     KiwixApplication.getApplicationComponent().inject(this);
   }
 
-  @Override
-  public int getCount() {
-    return mData.size();
+  @Override public Filter getFilter() {
+    return mFilter;
```

Comment:
No need to use Hungarian notations. I know they were already being used but we want to remove all of them. So you can refactor the code in this file to remove all the Hungarian notations.